cmd/fleetctl/convert.go

```diff
@@ -19,15 +19,60 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func convertPlatform(platformsIn string) (*string, error) {
+	// mappings based on https://github.com/osquery/osquery/blob/b87a4b5f1567415a72acd5ecd0e9e7ab75754959/tools/codegen/genwebsitejson.py#L38C18-L38C18
+	platformMapping := map[string][]string{
+		"darwin":    {"darwin"},
+		"linux":     {"linux"},
+		"windows":   {"windows"},
+		"chrome":    {"chrome"},
+		"specs":     {"darwin", "linux", "windows"},
+		"utility":   {"darwin", "linux", "windows"},
+		"yara":      {"darwin", "linux", "windows"},
+		"smart":     {"darwin", "linux"},
+		"kernel":    {"darwin"},
+		"linwin":    {"linux", "windows"},
+		"macwin":    {"darwin", "windows"},
+		"posix":     {"darwin", "linux"},
+		"sleuthkit": {"darwin", "linux"},
+		"any":       {""},
+		"all":       {""},
+		"":          {""},
+	}
+
+	resultOrder := []string{"darwin", "linux", "windows", "chrome"}
```

Comment:
Does that specific order really matter or is it just to have deterministic output? If it's the latter, you could use `sort.Strings(result)` at the end and do without that `resultOrder` slice.

Reply:
To document our discussion here, this order does not matter so I've made these changes